src/main/flight/imu.c

```diff
@@ -361,10 +361,10 @@ static bool imuIsAccelerometerHealthy(float *accAverage)
         accMagnitude += a * a;
     }
 
-    accMagnitude = accMagnitude * 100 / (sq((int32_t)acc.dev.acc_1G));
+    accMagnitude = sqrtf(accMagnitude) * acc.dev.acc_1G_rec;
 
-    // Accept accel readings only in range 0.90g - 1.10g
-    return (81 < accMagnitude) && (accMagnitude < 121);
+    // Accept accel readings only in range 0.80g - 1.20g
+    return (0.80f <= accMagnitude) && (accMagnitude <= 1.20f);
```

Comment:
(using <= with float is a bit strange in this context)

Reply:
Aggreed, I'll open a simple PR to optimize this little piece of code, as per my other comment

Reply:
Yes, result is indistinguishable. `<` seems a bit more readable to me (you don't have to think about the equality part when reading it)